pkg/applicationserver/io/web/templates.go

```diff
@@ -208,3 +223,86 @@ func applyWebhookTemplateFieldMask(dst, src *ttnpb.ApplicationWebhookTemplate, p
 	}
 	return dst, dst.SetFields(src, paths...)
 }
+
+type webhookTemplateField struct {
+	ID           string `yaml:"id"`
+	Name         string `yaml:"name"`
+	Description  string `yaml:"description"`
+	Secret       bool   `yaml:"secret"`
+	DefaultValue string `yaml:"default-value"`
+}
+
+func (f webhookTemplateField) toPB() *ttnpb.ApplicationWebhookTemplateField {
+	return &ttnpb.ApplicationWebhookTemplateField{
+		ID:           f.ID,
+		Name:         f.Name,
+		Description:  f.Description,
+		Secret:       f.Secret,
+		DefaultValue: f.DefaultValue,
+	}
+}
+
+type webhookTemplate struct {
+	TemplateID           string                 `yaml:"template-id"`
+	Name                 string                 `yaml:"name"`
+	Description          string                 `yaml:"description"`
+	LogoURL              string                 `yaml:"logo-url"`
+	InfoURL              string                 `yaml:"info-url"`
+	DocumentationURL     string                 `yaml:"documentation-url"`
+	BaseURL              string                 `yaml:"base-url"`
+	Headers              map[string]string      `yaml:"headers,omitempty"`
+	Format               string                 `yaml:"format"`
+	Fields               []webhookTemplateField `yaml:"fields,omitempty"`
+	CreateDownlinkAPIKey bool                   `yaml:"create-downlink-api-key"`
+	UplinkMessagePath    *string                `yaml:"uplink-message-path,omitempty"`
```

Comment:
We might want to group these under a `struct {...} ``yaml:"paths"`` ` for structure